src/components/ItemFilters/ItemFilters.tsx

```diff
@@ -37,6 +38,9 @@ const ItemFilters = ({
   appliedFilters = { location: [], format: [], status: [], year: [] },
   filtersAreApplied = false,
 }: ItemFilterContainerProps) => {
+  // We have to set the year value in state to be able to test form control in jest.
+  // TODO: Remove this if we can find a better way to test form submissions in jest.
+  const [year, setYear] = useState(appliedFilters.year[0] || "")
```

Comment:
This seems fine, I think we wanted to keep track of the text value as state for controlled components.